### PR TITLE
Read worksheet hidden state

### PR DESCRIPTION
**Pull request by LesterLyu**
## Fix bug: Cannot properly read/write worksheet.state
Related issues: https://github.com/exceljs/exceljs/issues/73 https://github.com/exceljs/exceljs/issues/226 https://github.com/exceljs/exceljs/issues/648

### CODE REVIEW

File: lib/doc/workbook.js [hunk 1]
```diff
@@ -193,10 +193,12 @@ Workbook.prototype = {
       var id = worksheetModel.id;
       var name = worksheetModel.name;
       var orderNo = value.sheets.findIndex(ws => ws.id === id);
+      var state = worksheetModel.state;
       var worksheet = this._worksheets[id] = new Worksheet({
         id: id,
         name: name,
         orderNo,
+        state,
```
Comment (LesterLyu) [inline lib/doc/workbook.js]: **Tthis two lines let exceljs able to read worksheet state.**

**Comment by Siemienik** [comment]
#621

**Comment by guyonroche** [comment]
@Siemienik - #621 is now merged - should we drop this one?

**Comment by Siemienik** [comment]
@guyonroche I think we should merge it, this PR contains test and additional changes in readme and workbook.js

**Pull request merged**